lib/active_merchant/billing/gateways/braintree_blue.rb

```diff
@@ -171,10 +171,6 @@ def unstore(customer_vault_id, options = {})
       end
       alias_method :delete, :unstore
 
-      def supports_network_tokenization?
```

Comment:
I'm curious to know why this was _always_ true. Does `Braintree Blue` always support network tokenization?